modular_skyrat/modules/sec_haul/code/misc/vending.dm

```diff
@@ -22,7 +22,7 @@
 		/obj/item/storage/fancy/donut_box = 2
 	)
 	premium = list(
-		/obj/item/storage/belt/security/webbing = 5,
+		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeepr
```

Comment:
```suggestion
		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeeper
```
typo moment

also the reason I'm doing a big review of my own file is bc its easier to sort through all the changes in this, so uh, dont mind me